Cargo.lock

```diff
@@ -136,7 +142,7 @@ dependencies = [
 
 [[package]]
 name = "configure"
-version = "0.5.0"
+version = "0.6.1"
```

Comment:
This was an oversight on my part during the Gradle Plugin updates - sorry about that!